idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -426,22 +427,22 @@ def rule_cp_cold(blk, t):
                           self.PH, initialize=390, units=pyunits.K,
                           doc="Cold Temperature OUT of pass")
 
-        # ======================================================================
+        # =====================================================================
         # PERFORMANCE EQUATIONS
```

Comment:
@andrewlee94 add to mass flow to property package (inkg/s)

Reply:
Mass flow should already be a property.